pom.xml

```diff
@@ -177,7 +177,7 @@ under the License.
 		<checkstyle.version>10.18.2</checkstyle.version>
 		<!-- can be removed with maven-spotless-plugin:2.38+ -->
```

Comment:
can we remove this comment and the line below now we are 2.38+

Reply:
removed.
>can we remove this comment now we are 2.38+ . Is there anything to action? I assume we can still skip spotless at 2.38+. Otherwise we should add this asap to minimize merge issues for open PRs.

Didn't get your comment.
The reason it was skipped for lower than 2.38 is the fact that spotless with those version didn't support java 21.
Regarding merge issues: I wouldn't expect lots of conflicts since changes are related to some weird situations (you can find them in changeset)

Reply:
thanks for the reassurance @snuyanzin  that there should be a low change of merge conflicts - I had not looked at the effected files in detail